champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/assassin/passives/ComboAttack.java

```diff
@@ -22,22 +19,25 @@
 import org.bukkit.event.Listener;
 import org.bukkit.event.entity.EntityDamageEvent.DamageCause;
 
+import javax.inject.Inject;
+import javax.inject.Singleton;
```

Comment:
Use com.google.inject imports instead of javax.inject

Reply:
The ones you removed basically

Reply:
whats the difference?

Reply:
It is no longer properly supported by guice, I changed all the imports to use the official google ones a while back due to occasional errors